test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval_expiration.test

```diff
@@ -30,75 +30,91 @@ accumulatorStart
 
 --SHELL--
 
-url="/v1/updateContext"
```

Comment:
Just cleanup.
Initially this test case didn't work and to understand what it did ... I had to fix it

NTC